docs/interaction_based_testing.adoc

```diff
@@ -681,6 +681,18 @@ subscriber.receive(_) >>> ["ok", "fail", "ok"] >> { throw new InternalError() }
 This will return `"ok", "fail", "ok"` for the first three invocations, throw `InternalError`
 for the fourth invocations, and return `ok` for any further invocation.
 
+
+=== Returning a default response
+
+If you don't really care what you return, but you must return a non-null value, you can use `_`.
+This will use the same logic to compute a response as `Stub` (see <<Stubs>>), so it is only really useful for `Mock` and `Spy` instances.
```

Comment:
As you found in the one test, there actually is a valid use-case for `Stub`s too, if you chain different responses and one of them should be the default one.
Would maybe make sense to mention this here and provide an example?